src/modules/domain/client.ts

```diff
@@ -56,16 +61,16 @@ export abstract class Client {
      * A better implmentation would be to save the (resolve,reject) in a list and
      * call them when the onStateChanged event happens.
      *
-     * @param pTimeoutMS optional number of MS to wait. Default is to 5 minutes.
-     * @returns the Client being waited on
-     * @throws exception if waited more than the timeout interval
+     * @param timeoutMS `(Optional)` The number of milliseconds to wait. Default is `300,000` (5 minutes).
```

Comment:
```suggestion
     * @param [timeoutMS] The number of milliseconds to wait. Default is `300,000` (5 minutes).
```
Using square brackets should work for indicating optional parameters.

Reply:
I've tried:
- @param [timeoutMS]
- @param [timeoutMS=]
- @param [timeoutMS=300000]
- @param {number} [timeoutMS]
- @param {number=} [timeoutMS]
- @param {number|undefined} [timeoutMS]

Unfortunately, none of these appear to mark the parameter as optional. All produce the same output:
![image](https://github.com/vircadia/vircadia-web/assets/22832983/153e64c5-ed16-457f-8c3d-306f35e4d8e2)

Is there a particular setting I should change for these to work?

Reply:
Hmmm, OK. Square brackets should work for TypeDoc / JSDoc, but I don't know about Intellisense conventions.
Something to keep a look out for in our travels.

Reply:
Roger, I'll keep an eye out.